src/node/routes/index.ts

```diff
@@ -127,10 +119,12 @@ export const register = async (
   app.use("/static", _static.router)
   app.use("/update", update.router)
 
-  const papi = new PluginAPI(logger, process.env.CS_PLUGIN, process.env.CS_PLUGIN_PATH)
+  const workingDir = args._ && args._.length > 0 ? path.resolve(args._[args._.length - 1]) : undefined
+  const papi = new PluginAPI(logger, process.env.CS_PLUGIN, process.env.CS_PLUGIN_PATH, workingDir)
```

Comment:
nit: I know this isn't a new addition from your PR but wish we could stick to actual names, like calling this `pluginAPI` instead of `papi` lol

Reply:
Hahaha I'm down for that.